### PR TITLE
Page 7 of developer documentation : Introduction for new developers

### PR DESCRIPTION
**Pull request by wbhart**
Should be ready for review.

### CODE REVIEW

File: docs/src/developer/introduction.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+This ticket is updated every so often.
+
+## Binaries
+
+Binaries of C libraries for Nemo are currently made in a separate repository:
+
+https://github.com/oscar-system/FLINT_jll.jl
+
+If code is added to any of the C libraries used by Nemo, this jll package must
+be updated first and the version updated Nemo.jl before the new functionality
```
Comment (rfourquet) [inline docs/src/developer/introduction.md:233]:
```suggestion
be updated first and the version updated in Nemo.jl before the new functionality
```

Comment (wbhart) [reply docs/src/developer/introduction.md]: Fixed.

File: docs/src/developer/introduction.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+https://github.com/Nemocas/AbstractAlgebra.jl/issues/492
+
+This ticket is updated every so often.
+
+## Binaries
+
+Binaries of C libraries for Nemo are currently made in a separate repository:
+
+https://github.com/oscar-system/FLINT_jll.jl
```
Comment (thofma) [inline docs/src/developer/introduction.md:230]: This is not the official version, but just some experiment to try a beta version of 2.7. The official build recipe resides at https://github.com/JuliaPackaging/Yggdrasil. This is the case for flint, arb and antic.

File: docs/src/developer/introduction.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+Within each of these directories is a set of files, one per module within
+the C libraries, e.g. the `fmpz.jl` file wraps the Flint `fmpz` module for
+multiple precision integers. The `fmpz_poly.jl` file wraps the Flint
+univariate polynomials over `fmpz` integers, and so on.
+
+The `fmpq` prefix is for Flint rationals, `fq` for Flint finite fields with
+multiprecision characteristic, `fq_nmod` is the same but for single word
+characteristic. The `padic` prefix is for the field of p-adic numbers for a
+given `p`. The `nmod` prefix is for Z/nZ for a given n. The `gfp` prefix is
```
Comment (thofma) [inline docs/src/developer/introduction.md:64]:
```suggestion
given `p`. The `nmod` prefix is for Z/nZ for a given `n`. The `gfp` prefix is
```

File: docs/src/developer/introduction.md (hunk 2 above)
Comment (wbhart) [reply docs/src/developer/introduction.md:230]: Fixed.

File: docs/src/developer/introduction.md (hunk 3 above)
Comment (wbhart) [reply docs/src/developer/introduction.md:64]: Fixed.

**Comment by wbhart** [comment]
Can this be merged?

**Comment by thofma** [comment]
I think the git setup is a bit non-standard, but we can add something else later.

**Pull request merged**